docs/antrea-proxy.md

```diff
@@ -253,6 +253,24 @@ data:
       proxyLoadBalancerIPs: false
 ```
 
+It's worth noting that AntreaProxy will bypass all the external `loadBalancerIP`s
+with the above configuration. After K8s v1.29, field `LoadBalancerIPMode` was
+introduced to `LoadBalancerIngress` in Service status, providing users with a
+more-fined mechanism to control how a corresponding external `loadBalancerIP` behaves.
+
+- If the value of `LoadBalancerIPMode` is `LoadBalancerIPModeVIP` or nil, the
+  traffic destined for the external `loadBalancerIP` should behave the same as
+  current implementation, getting load-balanced at the source Node.
+- If the value of `LoadBalancerIPMode` is `LoadBalancerIPModeProxy`, the traffic
+  destined for the external `loadBalancerIP` should be sent to the external loadBalancer.
+
+After Antrea v2.0, AntreaProxy respects `LoadBalancerIPMode` in Service status when
+`proxyLoadBalancerIPs` is set to `true`. In this case, AntreaProxy serves only the
+external `LoadBalancerIP`s configured with `LoadBalancerIPModeVIP`, bypassing those
```

Comment:
s/bypassing/ignoring

same below

The usage is not correct here IMO, because it is the Service traffic that bypasses AntreaProxy in this case, not AntreaProxy that bypasses the Service traffic.

Reply:
You are right. The external `LoadBalancerIP`s configured with `LoadBalancerIPModeProxy` will not be consumed by AntreaProxy, and AntreaProxy never has a chance to bypass them. For those configured with `LoadBalancerIPModeVIP `, they are consumed by AntreaProxy, and AntreaProxy can be able to decide to bypass all of them depending on the option `proxyLoadBalancerIPs`.